robotbase/src/main/java/com/team2357/frc2022/shuffleboard/AutoModeCommandChooser.java

```diff
@@ -0,0 +1,93 @@
+package com.team2357.frc2022.shuffleboard;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+// TODO: ShuffleBoard Implement
```

Comment:
```TODO```s can be removed once you do whatever they say to do, same with any comments describing the issue.